csharp/src/Ice/Internal/ObjectAdapterFactory.cs

```diff
@@ -7,7 +7,7 @@ public sealed class ObjectAdapterFactory
 {
     public void shutdown()
     {
-        List<Ice.ObjectAdapterI> adapters;
+        List<Ice.ObjectAdapter> adapters;
```

Comment:
We should clean-up Ice. qualification in this file.

Reply:
And many other files. That's for another PR.